.devops/code-review-pipelines.yml

```diff
@@ -0,0 +1,100 @@
+# Azure DevOps pipeline to build, check source codes and run tests.
+#
+# To make Danger JS run on a pull request you need to add the following pipeline 
+# variable and set it with a GitHub access token (scope public_repo); otherwise 
+# set its value to 'skip' without marking it secret: 
+# - DANGER_GITHUB_API_TOKEN
+# 
+
+variables:
+  NODE_VERSION: '10.14.1'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+
+# Automatically triggered on PR
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#pr-trigger
+trigger: none
+
+# Execute agents (jobs) on latest Ubuntu version.
+# To change OS for a specific, ovverride "pool" attribute inside the job definition
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+  - stage: Build
+    dependsOn: []
+    jobs:
+      - job: make_build  
+        steps:
+        - template: azure-templates/setup-project.yml
+        - script: |
+            yarn build
+          displayName: 'Build'
+  
+  - stage: Static_analysis
+    dependsOn: []
+    jobs:
+
+      - job: lint     
+        steps:
+        - template: azure-templates/setup-project.yml
+        - script: |
+            yarn lint
+          displayName: 'Lint'
+
+      - job: validate_api_specification     
+        steps:
+        - script: |
+            # Adding Oval is quicker than using npx multiple times
+            # Changes are not committes, so it's the same result
+            yarn add --dev oval
```

Comment:
This command changes the `package.json` and `yarn.lock` file. Are these changes ignored we a new package version is created?

Reply:
Yes, as every job runs on its own agent. 
As alternative, we could install `oval` as a dev depenedency, but that would require us to install all dependency just for this command (please note that we are not fetching dependency here)